picodom.d.ts

```diff
@@ -1,26 +1,41 @@
-export interface VNode<Data> {
+export as namespace picodom
+
+export interface VirtualNode<Data> {
```

Comment:
Rename `VNode` to `VirtualNode` per [PR 311 comment in hyperapp](https://github.com/hyperapp/hyperapp/pull/311)